Library/Homebrew/linkage_checker.rb

```diff
@@ -191,6 +194,12 @@ def check_dylibs(rebuild_cache:)
           @brewed_dylibs[f] << dylib
         end
       end
+
+      # Binary executables that link @rpath-prefixed dylibs must include at least one LC_RPATH.
+      @executables_missing_rpaths << file if
```

Comment:
Can we move this up into the block for handling `@variable_dylibs`?

Reply:
Done.